#!/usr/bin/env python
PACKAGE = "transbot_laser"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("linear", double_t, 0, "linear in PID", 0.3, 0, 0.45)
gen.add("angular", double_t, 0, "angular in PID", 1.0, 0, 2.0)
gen.add("LaserAngle", int_t, 0, "LaserAngle", 30, 10, 180)
gen.add("ResponseDist", double_t, 0, "ResponseDist", 0.55, 0, 8)
gen.add("switch", bool_t, 0, "switch in rosbot", False)
exit(gen.generate(PACKAGE, "laser_Avoidance", "laserAvoidPID"))
